chapters/2-document-creation-information.md

```diff
@@ -242,6 +330,36 @@ Example:
 
 NOTE In RDF, a namespace can be created for the external document reference if a short form name for the external reference is desired.
 
+**2.6.7** YAML: `externalDocumentRefs:`
+
+Example:
+
+```text
+externalDocumentRefs:
+  - externalDocumentId: "DocumentRef-spdx-tool-1.2"
+    spdxDocument: "http://spdx.org/spdxdocs/spdx-tools-v1.2-3F2504E0-4F89-41D3-9A0C-0305E82C3301"
+    - checksum:
+        algorithm: "SHA1"
+        checksumValue: "d6a770ba38583ed4bb4525bd96e50461655d2759"
```

Comment:
I think this should be:
```
  checksums:
    - algorithm: "SHA1"
      checksumValue: "d6a770ba38583ed4bb4525bd96e50461655d2759"
```

Reply:
Unless there is only one checksum in which case:
```
  checksum:
    algorithm: "SHA1"
    checksumValue: "d6a770ba38583ed4bb4525bd96e50461655d2759"
```
I am going by the JSON example below though.

Reply:
Since there can be only one checksum in the externalDocumentRef, I believe the example is correct (singular rather than plural).